source/sync/event-library.txt

```diff
@@ -0,0 +1,319 @@
+.. _realm-event-library:
+
+===================
+Realm Event Library
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+The Realm Event Library enables developers to track what data the user 
+sees and edits while using a Realm Sync-enabled mobile application. The 
+Event Library can record three types of events:
+
+- Read events
+- Write events
+- Custom events
+
+Developers can specify the read and write transactions they want to record.
+Additionally, you can configure custom events to record things like 
+button presses, or what the user is seeing in the frontend application.
+
+This level of detail enables auditors or other interested parties to
+replay exactly what happened, and when.
```

Comment:
nit: you don't really replay the when, you just know when. Maybe "know when changes were applied and replay them exactly as they happened."